scheduler/backend/backend.go

```diff
@@ -51,7 +51,7 @@ func (b Backends) AvailabilityZone(backendID string) (string, error) {
 			return backend.AvailabilityZone, nil
 		}
 	}
-	return "", errors.New(fmt.Sprintf("No backend with ID %s found", backendID))
+	return "", errors.New(fmt.Sprintf("No backend with GUID %s found", backendID))
```

Comment:
Yep, we should clarify "cell" = VM; and "node" = container. Might make this PR's specific changes difficult to consider in isolation; so will do in future PR.